test/index.js

```diff
@@ -74,10 +74,8 @@ describe('Lalamove API', () => {
     , secret: 'abc123'
     , country: 'SG'
     })
-    return lalamove.quotation(body).then(() => {
-      assert.equal('should not be able to call here', '')
-    }).catch((e) => {
-      assert.equal(e.status, 401)
+    return lalamove.quotation(body).then().catch((error) => {
+      assert.equal(error.response.status, 401)
     })
   })
 
```

Comment:
In axios library only http status 200 will consider as a success return otherwise all of it will goto error callback
Reference : https://github.com/mzabriskie/axios#handling-errors